cisc/lib.go

```diff
@@ -94,18 +96,16 @@ func (cfg *ciscConfig) proposeSendVoteUpdate(p *identity.Config) {
 	log.ErrFatal(cfg.ConfigUpdate())
 }
 
-// writes the ssh-keys to an 'authorized_keys'-file
+// writes the ssh-keys to an 'authorized_keys.cisc'-file
 func (cfg *ciscConfig) writeAuthorizedKeys(c *cli.Context) {
 	var keys []string
 	dir, _ := sshDirConfig(c)
-	authFile := dir + "/authorized_keys"
-	// Make backup
-	b, err := ioutil.ReadFile(authFile)
-	if err == nil {
-		err = ioutil.WriteFile(authFile+".back", b, 0600)
-		log.ErrFatal(err)
+	authFileOrig := filepath.Join(dir, "authorized_keys")
+	authFile := authFileOrig + ".cisc"
+	if _, err := os.Stat(authFileOrig); os.IsNotExist(err) {
+		log.Info("Making link from authorized_keys to authorized_keys.cisc")
+		os.Symlink(authFile, authFileOrig)
```

Comment:
Here you do a symlink automatically when the `authorized_keys.cisc` does not exists, is that right ?
But in that case I don't understand the comment above 
>  For safety reasons, this function saves to authorized_keys.cisc. But the sysadmin can make a link.

It seems like onet does the link automatically, and that we still do overwrite `authorized_keys` vanilla, since we write to the symlink ? 
There's something I don't get ...